android/src/main/java/com/pauldemarco/flutter_blue/FlutterBluePlugin.java

```diff
@@ -158,25 +165,22 @@ private void setup(
             if (registrar != null) {
                 // V1 embedding setup for activity listeners.
                 registrar.addRequestPermissionsResultListener(this);
-            } else {
-                // V2 embedding setup for activity listeners.
-                activityBinding.addRequestPermissionsResultListener(this);
```

Comment:
I'd like to understand this code more. Why is the V2 embedding removed?

Reply:
It was moved to onAttachedToActivity(ActivityPluginBinding binding).